inference/core/managers/active_learning.py

```diff
@@ -34,7 +35,14 @@ async def infer_from_request(
             model_id=model_id, request=request, **kwargs
         )
         active_learning_eligible = kwargs.get(ACTIVE_LEARNING_ELIGIBLE_PARAM, False)
-        if not active_learning_eligible or request.api_key is None:
+        active_learning_disabled_for_request = kwargs.get(
```

Comment:
Should this be default False to match the default value on the request entities?

Reply:
it can be both as parameter haas default value, but right - I will change into `false`